### PR TITLE
Update owner_cog.py

### PR DESCRIPTION
**Pull request by Emimendoza**
Added superior version of eval command

### CODE REVIEW

File: bot/cogs/owner_cog.py [hunk 1]
```diff
@@ -165,13 +165,13 @@ async def get(self, ctx, lines: int):
...
-            res = await res
-
-        
-        await ctx.send(res)
+        code = code.replace('```python', '')
+        code = code.replace('`', '')
+        code = code.replace('\n', '\n\t')
+        t=[None]
+        code = 'async def foobar():\n\t' + code + '\nt[0] = loop.create_task(foobar())'
+        exec(code, {'asyncio': asyncio, 'bot': self.bot, 'code': code, 'ctx': ctx, 'loop': asyncio.get_running_loop(), 'self': self, 't': t})
```
Comment (Jay-Madden) [inline bot/cogs/owner_cog.py:173]: lets pull this dictionary out into its own variable and also we dont need to pass in self here

File: bot/cogs/owner_cog.py [hunk 2]
```diff
@@ -165,13 +165,13 @@ async def get(self, ctx, lines: int):
...
-        code = code.strip('`')
-        res = eval(code)
-        if inspect.isawaitable(res):
-            res = await res
-
-        
-        await ctx.send(res)
+        code = code.replace('```python', '')
+        code = code.replace('`', '')
+        code = code.replace('\n', '\n\t')
```
Comment (Jay-Madden) [inline bot/cogs/owner_cog.py:170]: how does this handle spaces?

File: bot/cogs/owner_cog.py (hunk 1 above)
Comment (Emimendoza) [reply bot/cogs/owner_cog.py]: Dome

File: bot/cogs/owner_cog.py (hunk 2 above)
Comment (Emimendoza) [reply bot/cogs/owner_cog.py:170]: It can handle spaces as indents as long as its 5 spaces

**Review by Jay-Madden — APPROVED** [review]
Very well done ty

**Pull request merged**